.github/workflows/platform_tests-command.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [macos-latest, windows-latest, ubuntu-latest]
-        python: ["3.8", "3.9", "3.10"]
+        python: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
To trigger a platform test, you should be able to do it with comment on this PR by
`/platform_tests ref=ddelange:cp311`

However, our platform tests are currently failing because of other issues. @gradientsky @Innixma We'll want to fix the platform tests to unblock this PR